<launch>

  <node pkg="mech_ros" type="IMU_from_raw.py" name="converter" output="screen">
      <param name="calibrate_imu" value="False" />
      <param name="samples_number" value="1000" />
      <param name="calibrations_file" value="$(find mech_ros)/localization/IMU_calibration.yaml" />

  </node>

<!-- Taken from https://github.com/juancamilog/calibrate_imu.git and adjusuted for custom needs -->
  <node pkg="mech_ros" type="CalibrateIMU.py" name="calibrator" output="screen">
    <param name="publish_calibrated" value="false" />
    <param name="max_samples_mag" value="18000" />
    <param name="max_samples_imu" value="100000" />
   <param name="imu_topic" value="/imu/data_raw" />
   <param name="mag_topic" value="imu/mag" />
    <param name="calibrations_dir" value="$(find mech_ros)/bag_files/Mereni_IMU" />
  </node>
</launch>


